sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc

```diff
@@ -340,9 +384,35 @@ void joint_matrix_store_checked(Group g,
 The property list associated with the `annotated_ptr` argument
 represents the compile-time constant properties for cache control included
 in the SYCL extenion
-link:../../proposed/sycl_ext_intel_cache_controls.asciidoc[sycl_ext_intel_cache_controls],
-and for the alignment property defined in the SYCL extension
-link:../../experimental/sycl_ext_oneapi_kernel_arg_properties.asciidoc[sycl_ext_oneapi_kernel_arg_properties].
+link:../../proposed/sycl_ext_intel_cache_controls.asciidoc[sycl_ext_intel_cache_controls].
+
+==== Restrictions and Device Information Descriptors
+The following restrictions apply to these checked APIs:
+
+- The base pointer must be 4 bytes aligned.
+
+- For 8 bits data type, `CoordX` must be a multiple of 4. For 16 bits
+data type, `CoordX` must be a multiple of 2. So `CoordX` must be a
+multiple of 4 divided by size of the element type (`4/sizeof(T)`).
+
+- For 8 bits data type, `Width` must be a multiple of 4, For 16 bits
+data type, `Width` must be a multiple of 2. So `Width` must be a
+multiple of 4 divided by size of the element type (`4/sizeof(T)`).
```

Comment:
Describing it this way is confusing, and not really right.  For example, you first say that the base pointer must be 4 byte aligned, and then you say there is a query to get this alignment.  Presumably, we expect the alignment requirement to be different someday, so the spec should not say the requirement is 4 bytes.  I think you should remove these bullet points and say something like:

> Applications must adhere to certain alignment restrictions when using the checked APIs described in this section.  This extension provides the following queries to get these requirements:
>
> [Table]

The "Description" column will need to be expanded a bit to describe some of the details that are in the bullet points.  For example, the description of `matrix_checked_alignment` might say:

> Tells the required alignment (in bytes) of the base pointer for `joint_matrix_load_checked` and `joint_matrix_store_checked`.

If you want to document the alignment requirements of our current hardware, you could do that after the table describing the queries.  But you should somehow make it clear that the values only apply to our existing devices, and future devices may have different requirements.

Reply:
That's a good point, this is tying the API to a specific hardware which should be avoided.
yes, I will add the alignment requirements as an appendix like we did for the combinations table per each device. Here, these only apply to PVC for now.